lib/turbo/engine.rb

```diff
@@ -17,7 +17,7 @@ class Engine < Rails::Engine
     )
 
     initializer "turbo.no_action_cable" do
-      Rails.autoloaders.once.do_not_eager_load(Dir["#{root}/app/channels/turbo/*_channel.rb"]) unless defined?(ActionCable)
+      Rails.autoloaders.once.do_not_eager_load(Dir["#{root}/app/channels/turbo/*_channel.rb"]) if !defined?(ActionCable) && Rails.autoloaders.zeitwerk_enabled?
```

Comment:
Would this fix the problem? If zetiwerk is disable and action cable is not present, the action cable files would still be loaded. We need to teach the classic autoload to not load this code as well when zeitwerk is not enabled.

Reply:
Because the purpose of this fix is not to stop autoloading the code, but to fix breaking with `NoMethodError`.
Loading things what might be unnecessary is trivial, compared to completely breaking the classic-autoloader application.

And I couldn't find a way to

> teach the classic autoload to not load this code

, but do you have any idea on it?

Reply:
I think maybe `paths["app/channel"].clear`? I think that would fix in both modes and not need to check if zeitwerk is enabled.

Reply:
Thanks for the suggestion. The given one ended up with:

```
/path/to/turbo-rails/lib/turbo/engine.rb:22:in `block in <class:Engine>': undefined method `clear' for #<Rails::Paths::Path:0x00000001348f1198 @paths=["app/channels"], ...>
```

but I managed to write an implementation which works with both Zeitwerk and the classic autoloader.
Please kindly check.